packages/account-v2/src/components/base/WalletDropdown/WalletDropdown.scss

```diff
@@ -0,0 +1,146 @@
+/* stylelint-disable color-no-hex */
+
+.wallets-dropdown {
```

Comment:
I think you should have a genralize name 
Is it related to Wallets project or overall deriv-app

Reply:
The whole base folder is planned to be removed once we have proper components. So for now they are just copies

Reply:
@likhith-deriv I don't recommend using Wallets keyword in Accounts V2 package as it may create confusion.
Do reach out to other team members regarding the naming conventions and also if it is needed to keep a base component in the code.. best to get this from the @deriv/ui library.